src/handlers/shared/check-task-stale.ts

```diff
@@ -1,9 +1,19 @@
-export function checkTaskStale(staleTask: number, createdAt: string) {
-  if (staleTask !== 0) {
-    const days = Math.floor((new Date().getTime() - new Date(createdAt).getTime()) / (1000 * 60 * 60 * 24));
-    const staleToDays = Math.floor(staleTask / (1000 * 60 * 60 * 24));
-    return days >= staleToDays && staleToDays > 0;
+import ms from "ms";
+
+function calculateDaysDifference(date1: Date, date2: Date): number {
+  const millisecondsPerDay = ms("1d");
+  return Math.floor((date1.getTime() - date2.getTime()) / millisecondsPerDay);
+}
+
+export function checkTaskStale(staleTaskMilliseconds: number, createdAt: string): boolean {
+  if (staleTaskMilliseconds === 0) {
+    return false;
   }
 
-  return false;
+  const currentDate = new Date();
+  const createdDate = new Date(createdAt);
+  const daysSinceCreation = calculateDaysDifference(currentDate, createdDate);
+  const staleDaysThreshold = Math.floor(staleTaskMilliseconds / ms("1d"));
+
+  return daysSinceCreation >= staleDaysThreshold && staleDaysThreshold > 0;
```

Comment:
is there a reason to convert to days? just get milliseconds of both dates and see if its higher than `staleTaskMilliseconds`